pkg/sqlite/tag.go

```diff
@@ -236,7 +236,6 @@ func (qb *tagQueryBuilder) QueryForAutoTag(words []string) ([]*models.Tag, error
 	var args []interface{}
 
 	// always include names that begin with a single character
-	singleFirstCharacterRegex := "^[\\w][.\\-_ ]"
```

Comment:
Why remove these?

Reply:
This is replaced with `const singleFirstCharacterRegex = `^[\w\p{L}][.\-_ ]` in `performer.go`.